pkg/subcmd/integration.go

```diff
@@ -1,21 +1,83 @@
 package subcmd
 
 import (
+	"context"
 	"log/slog"
 
+	"github.com/redhat-appstudio/tssc-cli/pkg/chartfs"
+	"github.com/redhat-appstudio/tssc-cli/pkg/config"
 	"github.com/redhat-appstudio/tssc-cli/pkg/integrations"
 	"github.com/redhat-appstudio/tssc-cli/pkg/k8s"
+	"github.com/redhat-appstudio/tssc-cli/pkg/resolver"
 
 	"github.com/spf13/cobra"
 )
 
-func NewIntegration(logger *slog.Logger, kube *k8s.Kube) *cobra.Command {
+// disableProduct disables a product by its name in the configuration.
+// It searches for the product and updates its 'Enabled' field accordingly.
+func disableProduct(
+	ctx context.Context,
+	productName string,
+	cfg *config.Config,
+	kube *k8s.Kube,
+) error {
+	spec, err := cfg.GetProduct(productName)
+	if err != nil {
+		return err
+	}
+
+	spec.Enabled = false
+	if err := cfg.SetProduct(productName, *spec); err != nil {
+		return err
+	}
+
+	return config.NewConfigMapManager(kube).Update(ctx, cfg)
```

Comment:
nit: I'd move these lines into the `PersistentPostRunE` block, given it's only a few lines.